packages/databyss-ui/editor/slate/clipboard.js

```diff
@@ -160,3 +161,218 @@ export const isFragmentFullBlock = (fragment, document) => {
   }
   return false
 }
+
+/*
+        looks up the refId of the fragment and replaces it with an updated value, paste blocks can become stale when copying and pasting
+
+        this function takes a blockList, fragment, value and currentState and returns updated { blockList, fragment } 
+        */
+
+export const updateClipboardRefs = ({ blockList, fragment, sourceCache }) => {
```

Comment:
Please annotate parameter as a list, like we do in the API package. Describe the data types.
```
// @blockList    array of slate blocks
// @fragment    Slate value
...
```

Reply:
Also, the param list in the comment doesn't match the param list in the code...